components/perftest-capp/src/main/capp/CBRProxy_1.0.0/CBRProxy-1.0.0.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<proxy name="CBRProxy" startOnLoad="true" transports="https http" xmlns="http://ws.apache.org/ns/synapse">
```

Comment:
Add licence header for synapse configurations.

Reply:
Fixed in https://github.com/wso2/performance-ei/pull/9